sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+        * As far as possible, information should be based on research and fact gathering, rather than pure marketing or speculation.  
+
+#### End User Input Gathering (Inbound Communication)
+
+    * Gather useful end user input and feedback regarding expectations, pain points, primary use cases etc.
+
+    * Compile this into easily consumable reports and/or presentations to assist projects with feature design, prioritization, UX etc.
+
+#### Community Enablement
+
+    * SIGs are open organizations with meetings, meeting agendas and notes, mailing lists, and other communications in the open
+
+    * The mailing list, SIG meeting calendar, and other communication documents of the SIG will be openly published and maintained
+
+#### As Trusted Expert Advisors to the TOC
+
+    * Perform technical due diligence on new and graduating projects, and advise TOC on findings.
+
+    * Be involved with, or periodically check in with projects in their area, and advise TOC on health, status and proposed actions (if any) as necessary or on request.
+
+#### SIG Charter:
+
+    * formally reviewed annually, and approved by the TOC.  The charter must clearly articulate:
+
+        * what is in and out of scope of the SIG
+
+        * whether and how it overlaps and interfaces with other CNCF SIG’s or other relevant groups.
+
+        * how it operates and is governed, and specifically whether and how it deviates from standard SIG operating guidelines provided by the TOC.  Deviation from these guidelines is discouraged, unless there are good and well-documented reasons for such divergence, approved by the TOC. 
+
+See [Example Responsibilities of a CNCF SIG](https://docs.google.com/document/d/1L9dJl5aBFnN5KEf82J689FY0UtnUawnt9ooCq8SkO_w/edit?usp=sharing).
+
+## Operating Model
+
+*Important*
+
+*Each SIG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Exec, plus communication and performance of the SIG, with quarterly and annual reporting to GB & TOC. *** **
+
+As a starting point let’s be inspired by CNCF OSS Projects and by K8S SIGs.  That means minimal viable governance and community-based organisation.
+
+### SIG Formation, Leadership and Membership Composition
+
+1. SIGs are formed by the TOC.  Initial SIGs are listed below, and will be adapted over time as required.  If members of the community believe that additional SIGs are desired, they should propose these to the TOC, with clear justification, and ideally volunteers to lead the SIG. The TOC wishes to have the smallest viable number of SIGs, and for all of them to be highly effective (as opposed to a "SIG sprawl" with large numbers of relatively ineffective SIGS).
+
+2. SIG has three co-chairs, who are TOC Contributors and recognised as unbiased experts in that area.
```

Comment:
References to "unbiased" (here and in # 4) seem problematic. Do they not automatically exclude project maintainers & tech leads? It is important that the SIG overall is unbiased, through diversity, not the individual members.

Reply:
Good question.  Let me try to rework the wording accordingly.  Ideally the individuals would themselves also be capable of being relatively unbiased.   As an extreme and contrived counter-example, I don't think that we want a diversity of highly biased and warring co-chairs or leads.  I'll try to capture that in words.

Reply:
I reworded as follows:
"Chairs... recognized as experts in that area, and for their ability to co-lead the SIG to produce the required unbiased outputs."

"Tech leads... demonstrating the ability to provide the balanced technical leadership required to produce the required unbiased outputs of the SIG."